---
layout: page
title: About me 
---

I'm Yuxuan Yuan and my English name is Andy

<p>
I am a computational biologist/bioinformatican working on genomics and bioinformatics
<p>
My research mainly focuses on genome assembly, genetic variation and pop-gen studies

<p>
The species that I have been working on are canola, wheat and soybean

<h3> Education Background </h3>

<p>
2015 - 2019, PhD, the University of Western Australia, Perth, Australia

<p>
2012 - 2014, Master, the University of Queensland, Brisbane, Australia

<p>
2007 - 2011, Bachelor, Zhejiang Chinese Medical University, Hangzhou, China

<p>

<h3> Recent Posts </h3>

<p>
2021 - present, Assistant Researcher (by courtesy), the Chinese University of Hong Kong, Shenzhen, China

<p>
2019 - present, Postdoctoral fellow, the Chinese University of Hong Kong, Hong Kong, China

<p>

<h3> Peer-review services </h3>

<p>
Topic editor for <i> Sustainability</i>

<p>
<i>Ad-hoc</i> reviewer for <i> Current Opinion in Plant Biology, GigaScience, International Journal of Molecular Sciences, BMC Plant Biology, Agronomy, Diversity, Plants </i> and <i>Cytometry Part A</i>

<p>

<h3> Publications </h3>  

<p>

* Bayer PE, Valliyodan B, Hu H, Marsh J, <i>Yuan Y</i> Vuong T, Patil G, Song Q, Batley J, Varshney R, Lam HM, Edwards D and Nguyen H (2021) Sequencing the USDA core soybean collection reveals gene loss during domestication and breeding, doi: 10.1002/tpg2.20109, the Plant Genome [IF: 4.089; 5-Year IF:4.972; JCR Plant Sciences: 42/235 (Q1)]

<p>

* <i>Yuan Y</i>, Bayer PE, Batley J and Edwards D (2021) Current status of structural variation studies in plants. Plant Biotechnology Journal. doi:10.1111/pbi.13646 [<b>IF: 9.803</b>; 5-Year IF: 9.555; JCR Plant Sciences: 8/235 (Q1)]

<p>

* <i>Yuan Y</i>, Chung CKL and Chan TF (2020), Advances in optical mapping for genomic research. Computational and Structural Biotechnology Journal, doi: 10.1016/j.csbj.2020.07.018 [<b>IF: 7.271</b>; 5-Year IF: 7.409; JCR BIOCHEMISTRY & MOLECULAR BIOLOGY: 45/298 (Q1)]

<p>

* Hu H, <i>Yuan Y</i>, PE Bayer, CT Fernandez, A Scheben, AA Golicz and Edwards D. (2020) Legume Pangenome Construction Using an Iterative Mapping and Assembly Approach, Legume Genomics, 35-47, doi: 10.1007/978-1-0716-0235-5_3

<p>

* Anderson R, CT Fernandez, <i>Yuan Y</i>, AA Golicz, D Edwards, PE Bayer (2020) Method for Genome-Wide Association Study: A Soybean Example. Legume Genomics, 147-158, doi: 10.1007/978-1-0716-0235-5_7

<p>

* Valliyodan B, Cannon S, Bayer B, Shu S, Ren L, Jenkins J, Chung C, Chan T, Daum C, Plott C, Hastie A, Baruch K, Barry W, Huang W, Patil G, Varshney R, Hu H, Batley J, <i>Yuan Y</i>, Song Q, Goodstein D, Stacey G, Lam H, Jackson S, Schmutz J, Grimwood J, Edwards D, Nguyen H (2019) Construction and comparison of three new reference-quality genome assemblies for soybean, the Plant Journal, doi: 10.1111/tpj.14500 [<b>IF: 6.417</b>; 5-Year IF: 7.627; JCR Plant Sciences: 15/235 (Q1)]

<p>

* <i>Yuan Y</i>, Scheben A, Batley J and Edwards D. (2019) Using genomics to adapt crops to climate change. In: Sarkar A., Sensarma S., vanLoon G. (eds) Sustainable Solutions for Food Security, Springer, Cham, doi:10.1007/978-3-319-77878-5_5

<p>

* <i>Yuan Y</i>, Bayer P, Vrana, J, Dolezel J, Edwards D, Erskine W, Kaur P. (2018) Large-scale structural variation detection in subterranean clover subtypes using optical mapping, Frontiers in Plant Science, 9, 971, doi: 10.3389/fpls.2018.00971 [<b>IF: 5.753 </b>; 5-Year IF: 6.612; JCR Plant Sciences: 17/235 (Q1)]

<p>

* <i>Yuan Y</i>, Lee H, Hu H, Scheben A, Edwards D. (2018) Single-cell genomic analysis in plants. Genes. doi: 10.3390/genes9010050 [<b>IF: 4.096 </b>; 5-Year IF: 4.339; JCR Genetics & Heredity: 65/175 (Q2)]

<p>

* <i>Yuan Y</i>, Scheben A, Chan CKK and Edwards D. (2017) Databases for wheat genomics and crop improvement. Wheat Biotechnology: Methods and Protocols, Methods in Molecular Biology, vol. 1679, doi: 10.1007/978-1-4939-7337-8_18, Springer      

<p>

* <i>Yuan Y</i>, Bayer P, Scheben A, Chan CKK, Edwards D. (2017) BioNanoAnalyst: A visualisation tool to assess genome assembly quality using BioNano data. BMC Bioinformatics. doi: 11.1186/s12859-017-1735-4 [<b>IF: 3.169</b>; 5-Year IF: 3.629; JCR Mathematical & Computational Biology: 16/58 (Q2)].

<p>

* <i>Yuan Y</i>, Bayer P, Lee H, Edwards D. (2017) runBNG: A software package for BioNano genomic analysis on the command line. Bioinformatics. doi: 10.1093/bioinformatics/btx366 [<b>IF: 6.937</b>; 5-Year IF: 8.47; JCR Mathematical & Computational Biology: 3/58 (Q1)]

<p>

* Bayer P, Hurgobin B, Golicz A, Chan K, <i>Yuan Y</i>, Lee H, Renton M, Meng J, Li R, Long Y, Zou J, Bancroft I, Chalhoub B, King G, Batley J, Edwards D. (2017) Assembly and comparison of two closely related Brassica napus genomes. Plant Biotechnology Journal. doi:10.1111/pbi.12742 [<b>IF: 9.803</b>; 5-Year IF: 9.555; JCR Plant Sciences: 8/235 (Q1)]

<p>

* <i>Yuan Y</i>, Bayer P, Batley J, Edwards D. (2017) Improvements in genomic technologies: application to crop genomics. Trends in Biotechnology. doi: 10.1016/j.tibtech.2017.02.009 [<b>IF: 19.536</b>; 5-Year IF: 19.599; JCR Biotechnology &Applied Microbiology: 3/159 (Q1)]

<p>

* Kaur P, Bayer P, Milec Z, Vrána J, <i>Yuan Y</i>, Appels R, Edwards D, Batley J, Nichols P, Erskine W, Doležel J. (2017) An advanced reference genome of Trifolium subterraneum L. reveals genes related to agronomic performance. Plant Biotechnology Journal. doi: 10.1111/pbi.12697 [<b>IF: 9.803</b>; 5-Year IF: 9.555; JCR Plant Sciences: 8/235 (Q1)]

<p> 

* Scheben A, <i>Yuan Y</i>, Edwards D. (2016) Advances in genomics for adapting crops to climate change. Current Plant Biology. 6: 2-10. doi:10.1016/j.cpb.2016.09.001


<h3> Preprints</h3>

<p>

* <i>Yuan Y</i>, Bayer P, Vrana, J, Dolezel J, Edwards D, Erskine W, Kaur P. (2017) Large-scale structural variation detection in subterranean clover subtypes using optical mapping validated at nucleotide level, bioRxiv, 232132, doi: 10.1101/232132

<p> 

* <i>Yuan Y</i>, Bayer PE, Anderson R, Lee HT, Chan CK, Zhao R, Batley J and Edwards D (2020), RefKA: A fast and efficient long-read genome assembly approach for large and complex genomes. bioRxiv, doi: 10.1101/2020.04.17.035287

<h3> Conferennce Proceedings</h3>

<p>

* <i>Yuan Y</i> and Chan TF (2020) panBuilder: a new pangenome construction pipeline using long-read sequencing and optical mapping. The International Plant & Animal Genome XXVIII, San Diego, USA, PO1063

<p>

* <i>Yuan Y</i>, Bayer P, Edwards D. (2016) Improving crop genome assemblies using optical mapping and long read sequencing. Brassica Conference 2016 Melbourne, PO053

<p>

* <i>Yuan Y</i>, Anderson R, Bayer P, Lee H, Chan C, Edwards D (2019) A fast and efficient long-read genome assembly approach for large and complex genomes. The International Plant & Animal Genome XXVII, San Diego, USA, PO0061

<h3>Conference presentations</h3>

<p>

* <i>Yuan Y</i>, Bayer P, Lee H, Scheben A, Chan C, Edwards D (2019) Bionano data analysis on the command line and genome assembly quality assessment using a GUI. The International Plant & Animal Genome XXVII, San Diego, USA, C01

<p>

<h3> Online resources</h3>

<p>

My publication record is on <a target="_blank" href='https://scholar.google.com.au/citations?user=2oLXfwoAAAAJ&hl=en'>GoogleScholar</a> 

<p> 

My CV is <a target="_blank" href='https://www.dropbox.com/s/dr5syy4vb05wn7z/CV_AY.pdf?dl=0'>here</a>

<p>

ORCID: <a target="_blank" href='https://orcid.org/0000-0003-0741-4196'>0000-0003-0741-4196</a>

<p>

My email: yuxuan.yuan@outlook.com; yuxuan.yuan@cuhk.edu.hk

<p>

<p>

Please contact me if you have any question regarding me or my page

<p> 

<p> 


{% include comments.html %}

